src/Microsoft.Framework.Caching.SqlServer/SqlQueries.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    internal class SqlQueries
+    {
+        private const string CreateTableFormat = "CREATE TABLE [{0}](" +
```

Comment:
Also take into account the memory optimized tables supported on recent versions of SQL Server. I believe it could give the cache a significant boost. The `CREATE TABLE` statement will look different and it might also give you another reason to consider stored procedures, which can be compiled to native and may give you another significant boost (once more, you would need to try and measure the impact). See https://msdn.microsoft.com/en-us/library/dn133079.aspx.